templates/security.html.twig

```diff
@@ -110,6 +137,24 @@
 
 {% block javascripts %}
     <script src="{{ asset('bundles/tabler/tabler.js') }}?{{ tabler_asset_version() }}"></script>
+
+    <script>
+        function toggleShowPasswordInput() {
+            const input = document.querySelector('input[name="_password"]')
```

Comment:
Okay, I am sorry about that code style nit-picking, but I missed it before.

Please add `;`. 
I not they are not necessary in JS world, but I already ran into bugs due to missing semicolons. 

I assume you have tested your changes, so I can safely merge?!

Reply:
> Okay, I am sorry about that code style nit-picking, but I missed it before.
> 
> Please add ;. I know that they are not necessary in JS world, but I already ran into bugs in the past due to missing semicolons.

It's been a long day since I wrote semicolons for JS/TS, but no problem 😃 

> I assume you have tested your changes, so I can safely merge after that one change?!

Yep 👍 

![Login](https://github.com/user-attachments/assets/c6e88d51-4bbd-4f7c-aa8c-d1f3f2f682b0)